packages/openapi-fetch/src/index.d.ts

```diff
@@ -10,10 +10,14 @@ import type {
   SuccessResponse,
 } from "openapi-typescript-helpers";
 
+import type { Dispatcher } from "undici";
```

Comment:
Can we redeclare this type so we don’t have an `undici` dependency (even type-wise)? The type signatures can match, but I don’t want people who install `openapi-fetch` to also have undici install as well.